core/tests/protractor/collections.js

```diff
@@ -82,6 +115,58 @@ describe('Collections', function() {
     users.logout();
   });
 
+  it('checks for console errors in a collection with one node', function() {
+    users.login('player@collections.com');
+    browser.get('/collection/' + collectionId);
+    browser.waitForAngular();
+    users.logout();
+  });
+
+  it('checks for console errors in a collection with two nodes', function() {
+    users.login('player@collections.com');
+    browser.get('/collection_editor/create/' + collectionId);
+    browser.waitForAngular();
+    collectionEditor.addExistingExploration('4');
+    collectionEditor.saveDraft();
+    collectionEditor.setCommitMessage('Add Exploration');
+    collectionEditor.closeSaveModal();
+    browser.waitForAngular();
+
+    browser.get('/collection/' + collectionId);
+    browser.waitForAngular();
+    users.logout();
+  });
+
+  it('checks for console errors in a collection with three nodes', function() {
+    users.login('player@collections.com');
+    browser.get('/collection_editor/create/' + collectionId);
+    browser.waitForAngular();
+    collectionEditor.addExistingExploration('13');
+    collectionEditor.saveDraft();
+    collectionEditor.setCommitMessage('Add Exploration');
+    collectionEditor.closeSaveModal();
+    browser.waitForAngular();
+
+    browser.get('/collection/' + collectionId);
+    browser.waitForAngular();
+    users.logout();
+  });
+
+  it('checks for console errors in a collection with four nodes', function() {
```

Comment:
It looks like these individual tests are not independent of each other. If that's the case, I suggest combining them all into a single, larger, test, that deals entirely with the same exploration.